src/components/semantic/Inserter.tsx

```diff
@@ -31,6 +31,7 @@ const blockTypes = {
     "image": {type: "image", encoding: "markdown", value: ""},
     "video": {type: "video", encoding: "markdown", src: "https://www.youtube.com/watch?v=<video_id>"},
     "tabs": {type: "content", layout: "tabs", encoding: "markdown", children: []},
+    "codeTabs": {type: "codeTabs", encoding: "markdown", children: []},
```

Comment:
We don't use camelCase for the other types, best to keep it consistent.
```suggestion
    "code tabs": {type: "codeTabs", encoding: "markdown", children: []},
```